src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,56 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie
+};
+
+export const MovieCards: React.FC<Props> = ({ movie }) => {
```

Comment:
The component name 'MovieCards' should be 'MovieCard' to match the file name.

Reply:
Agree